examples/deployment/src/main/yaml/example-cluster.yaml

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at
+# http://oss.oracle.com/licenses/upl.
+#
+# Example 1
+# Initial cluster with one role of 'storage`
+apiVersion: coherence.oracle.com/v1
+kind: CoherenceCluster
+metadata:
+  name: example-cluster
+spec:
+  imagePullSecrets:
+    - name: coherence-example-secret
+  jvm:
+    heapSize: 512m
+  ports:
+    - name: metrics
+      port: 9612
+  coherence:
+    logLevel: 5
```

Comment:
Coherence default log level is already `5`

Reply:
fixed